chainer/utils/argument.py

```diff
@@ -8,8 +8,12 @@ def parse_kwargs(kwargs, *name_and_values):
     values = [kwargs.pop(name, default_value)
               for name, default_value in name_and_values]
     if kwargs:
-        args = ', '.join(["'%s'" % arg for arg in kwargs.keys()])
-        raise TypeError('got unexpected keyword argument(s) %s' % args)
+        import inspect
```

Comment:
Please move `import` to the top of the file.

Reply:
I consider that this exception is raised so rarely that importing `inspect` module at top of the file might incur additional performance overhead, because [`import inspect`](https://github.com/chainer/chainer/search?utf8=%E2%9C%93&q=%22import+inspect%22&type=) is nearly zero used in the chainer core codes, in contrast with [`parse_kwargs`](https://github.com/chainer/chainer/search?utf8=%E2%9C%93&q=parse_kwargs&type=) used more frequently.